.circleci/config.yml

```diff
@@ -143,6 +143,7 @@ commands:
             # that will also block dependencies from being installed.
             # as a result we first install dependencies by installing the
             # package and then immediately remove it again
+            pip install --upgrade pip
```

Comment:
where did you end up re updating the `pip` version in `requirements-dev.txt` instead?

Reply:
I've put that into the `./configure.sh` and saves the hassle for `build` and `test` but for the pip package test I don't know whether we can put that pip upgrade somewhere else other than the .circleci config.

Reply:
would it be enough to update this line? https://github.com/tf-encrypted/tf-big/blob/master/requirements-dev.txt#L4

Reply:
In the case `whltest` needs to pull from `requirements-dev.txt` which does not since it wants to see whether a clean installation of the `tf_big` package using `pip` will work (hence no requirements in `requirements-dev.txt`)

Reply:
could we put the version requirement on pip for whltest in `setup.py`?

cc @jvmncs I remember we talked setup and best practices a while back, including having multiple requirements files. I can't seem to find it now but maybe you have a better memory/index..?

Reply:
This can be solved by upgrading the pip version on the docker I think?

Reply:
Even if we put that requirement in `setup.py` we will still need to install the newest version of `pip` using the .circleci config.